recipes/platform.interfaces/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.interfaces"
+    license = "The Unlicense"

```

Comment:
```suggestion
    license = "Unlicense"
```

Use SPDX format: https://spdx.org/licenses/Unlicense